docs/src/main/sphinx/installation/cli.rst

```diff
@@ -80,6 +80,35 @@ The detailed behavior is as follows:
   authentication system configuration.
 * Expired tokens force you to log in again.
 
+.. _cli-certificate-auth:
+
+Certificate authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use the following command line arguments to allow you to connect the CLI to a
```

Comment:
arguments to connect  to a cluster that uses certificate authentication with the CLI:

Reply:
OK, but with CLI at the end, it could be read as "a cluster ... with the CLI", meaning a cluster that has the CLI. I moved "CLI" back.